modules/phase_field/examples/multiphase/DerivativeMultiPhaseMaterial.i

```diff
@@ -0,0 +1,394 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 40
+  ny = 40
+  nz = 0
+  xmin = -12
+  xmax = 12
+  ymin = -12
+  ymax = 12
+  elem_type = QUAD4
+[]
+
+[GlobalParams]
+  # let's output all material properties for demonstration purposes
+  outputs = exodus
+
+  # prefactor on the penalty function kernels. The higher this value is, the
+  # more rigorously the constraint is enforced
+  penalty = 1e3
+[]
+
+#
```

Comment:
As soon as a user opens this in peacock and then saves it, all of the comments outside of the blocks will be stuck at the top of the input file. You need to move the comments describing a block inside of the block itself for peacock to not reformat it.